docs/database/cosmosdb.md

```diff
@@ -265,11 +271,155 @@ and SHOULD be provided **at span creation time** (if provided at all):
 | `db.operation.name`                  | `"read_item"` |
 | `server.address`                     | `"account.documents.azure.com"` |
 | `db.cosmosdb.client_id`              | `"3ba4827d-4422-483f-b59f-85b74211c11d"` |
-| `db.cosmosdb.operation_type`         | `"read"` |
 | `user_agent.original`                | `"cosmos-netstandard-sdk/3.23.0\|3.23.1\|1\|X64\|Linux 5.4.0-1098-azure 104 18\|.NET Core 3.1.32\|S\|"` |
 | `db.cosmosdb.request_content_length` | `20` |
-| `db.cosmosdb.status_code`            | `201` |
+| `db.response.status_code`            | `201` |
 | `db.cosmosdb.sub_status_code`        | `0` |
 | `db.cosmosdb.request_charge`         | `7.43` |
 
+## Operation Level Metrics
+
+The following metrics provide insights into Azure Cosmos DB client operation performance and behavior.
+
+### Metric: `db.client.operation.duration`
+
+This metric is [required][MetricRequired].
+
+It captures the total time taken by an Azure Cosmos DB operation. This metric follows the common [db.client.operation.duration](/docs/database/database-metrics.md#metric-dbclientoperationduration) definition.
+
+Refer [db.cosmosdb.operation.request_charge](#metric-dbcosmosdboperationrequest_charge) metrics for dimensions.
+
+### Metric: `db.client.response.row_count`
+
+This metric is [required][MetricRequired].
+
+It captures the number of items returned by a query or feed operation in Azure Cosmos DB. It helps identify response sizes that may contribute to high latency, increased memory/CPU usage, or network call failures. This metric follows the common [db.client.response.row_count](/docs/database/database-metrics.md#metric-dbclientresponserow_count) definition.
+
+Refer [db.cosmosdb.operation.request_charge](#metric-dbcosmosdboperationrequest_charge) metrics for dimensions.
+
+### Metric: `db.cosmosdb.operation.request_charge`
+
+This metric is [required][MetricRequired].
+
+It captures the Request Units consumed by each operation in Azure Cosmos DB. Since Request Units serve as a form of currency within the Azure Cosmos DB database, monitoring their usage is crucial to avoid throttling.
```

Comment:
```suggestion
It captures the Request Units consumed by each operation in Azure Cosmos DB. Since Request Units serve as a form of throughput control within the Azure Cosmos DB database, monitoring their usage is crucial to avoid throttling.
```